app/views/rubygems/show.html.erb

```diff
@@ -13,6 +13,12 @@
 
 <div class="l-overflow">
   <div class="l-colspan--l colspan--l--has-border">
+    <% if @latest_version.vulnerable %>
+      <div class="gem__vulnerable">
+        <p>This version is known to have security vulnerabilities.</p>
```

Comment:
Do list the advisory-db link as displaying a gem as merely vulnerable is not sufficient. Eg. in case of `nokogiri` gem, while disclosing a vulnerability often a patch update is provided prior to the vulnerability being fixed in upcoming release.

Reply:
Not all advisories will be from the ruby-advisory-db. Some are reported by the gem authors as well, so there won't be a link available. We can maybe display the link for the advisories reported via ruby-advisory-db and just display this message for those that are reported via terminal.
or
Since we storing all advisories in the rubygems database we can have a separate `gem#show/advisory` page to list advisories for a gem. A version can contain multiple advisories for example `actionpack-2.3.2` contains a total of 21 advisories.